style/style.css

```diff
@@ -0,0 +1,302 @@
+  text-decoration: none; }
+  /* line 199, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  .btn:hover {
+    background: #29c2ff;
+    background-image: -webkit-linear-gradient(top, #29c2ff, #1e97b8);
+    background-image: -moz-linear-gradient(top, #29c2ff, #1e97b8);
+    background-image: -ms-linear-gradient(top, #29c2ff, #1e97b8);
+    background-image: -o-linear-gradient(top, #29c2ff, #1e97b8);
+    background-image: linear-gradient(to bottom, #29c2ff, #1e97b8);
+    text-decoration: none; }
+
+/* I used the according @ http://codepen.io/thirdtiu/pen/uDyxr */
+/* line 212, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+.accordion {
+  background: #eeeeee; }
+  /* line 214, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  .accordion dt, .accordion dd {
+    padding: 7px 9px; }
+    /* line 216, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+    .accordion dt a, .accordion dd a {
+      display: block; }
+  /* line 220, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  .accordion dt {
+    border-top: 5px solid white; }
+    /* line 222, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+    .accordion dt h3 {
+      line-height: 1.3em;
+      margin: 0; }
+  /* line 227, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  .accordion dd {
+    border-top: none;
+    padding-top: 0; }
+    /* line 230, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+    .accordion dd:last-of-type {
+      position: relative;
+      top: -1px; }
+    /* line 234, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+    .accordion dd p {
+      font-size: .8em;
+      line-height: 1.4em; }
+
+/* The arrows on the FAQ accordion is something I didn't get to. It's been a long day. */
+/* line 242, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+#faq h2 {
+  margin-bottom: 17px; }
+
+@media (max-width: 910px) {
+  /* line 249, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  nav a:link, nav a:visited {
+    font-size: 0.7em;
+    line-height: 1em; } }
+
+@media (max-width: 800px) {
+  /* line 257, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  nav li {
+    height: 60px;
+    width: 100%;
+    float: none;
+    margin-right: 0px;
+    margin-bottom: 5px; }
+  /* line 264, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  nav img {
+    width: 7%; }
+  /* line 268, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  nav a:link, nav a:visited {
+    font-size: 0.9em; }
+  /* line 271, /Users/jonandmel/Documents/JONS WEB DESIGN/GitHub/fed-test/style/style.scss */
+  header .teaser_content p {
+    display: none; }
```

Comment:
Honestly, this is a decision I made to save time on the overall test. At a smaller device size, the text was getting lost in the background image. What I probably would have done is find a different type treatment in order for the text to be readable on a small device.